core/server/common/src/main/java/alluxio/cli/validation/HdfsValidationTask.java

```diff
@@ -50,13 +48,26 @@ public HdfsValidationTask() {}
 
   @Override
   public boolean validate(Map<String, String> optionsMap) {
+    if (shouldSkip()) {
```

Comment:
Since this method is defined on the interface, it would make more sense for the ValidateEnv class to call this method and skip the task instead of doing it in each task.

Reply:
Done.